src/main/resources/migration/common/V17__soft_delete_csip_record.sql

```diff
@@ -0,0 +1,240 @@
+truncate csip_record cascade;
+drop table attendee;
+drop table review;
+drop table identified_need;
+drop table interview;
+drop table contributory_factor;
+drop table plan;
+drop table decision_and_actions;
+drop table investigation;
+drop table safer_custody_screening_outcome;
+drop table referral;
```

Comment:
Apologies for the lack of domain knowledge here but this seems scary. Are we okay to drop tables at this point because we're not live in production? How often/when do migrations run? What are our plans to get data back for our test envs?

Reply:
Sweet, thanks just wanted to double check :)